app/assets/javascripts/modules/track-click.js

```diff
@@ -8,16 +8,26 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       element.on('click', trackClick);
 
       var options = {},
-          category = element.attr('data-track-category'),
-          action = element.attr('data-track-action'),
-          label = element.attr('data-track-label');
+          category = element.data('track-category'),
```

Comment:
As discussed, custom dimension should remain data because Google Analytics expects an integer.